arithmetic-formatter/arithmetic_arranger.py

```diff
@@ -0,0 +1,43 @@
+def arithmetic_arranger(problems, displayMode=False):
+    first_operand_line = second_operand_line = dashes_line = ""
+    result_line = ""
+
+    for index in range(len(problems)):
+        problem = problems[index]
+        [operand1, operator, operand2] = problem.split()
+
+        # Error Handling
+        if len(problems) > 5:
```

Comment:
We should put this before for loop else it will get executed on each loop with no difference.